scripts/zones/Crawlers_Nest/mobs/Drone_Crawler.lua

```diff
@@ -20,15 +20,28 @@ end;
 
 function onMobDeath(mob,killer)
 
-	mob = mob:getID();
-
-	if (mob == 17584131) then
-		GetNPCByID(17584457):hideNPC(900); -- qm3
-	elseif (mob == 17584132) then
-		GetNPCByID(17584458):hideNPC(900);-- qm4
-	end
+    mob = mob:getID();

```

Comment:
Thats actually a bad practice, and also wasn't local'd. Better to just say that bit of code twice than to var it as "mob" when mob is already the argument name of the function if only so future editors/readers don't get confused. 

General rule of thumb: 
Variablize stuff that is either changing farther down the line in your code, or that is annoying to read (like really long like like this: `GetMobByID(12345678):getCurrentAction() == ACTION_ANNOY_TEOS_EYES` because right/left scrolling or word wrapping code is "TEH DEBIL!"  I tell you) and *good* code rarely gets very "wide" anyway.

This is only used twice in the same script, and its pretty short. it's ok to say `Mob:getID()` twice here.